### PR TITLE
[OB 2.0] [AU] Add Testgrid yaml for AU staging environment

### PR DESCRIPTION
**Pull request by AmilaSamith**
## Task performed
- [ ] Added a new job
- [ ] Removed a job -> inform tg folks to clean the dashboard
- [ ] Added new infrastructure value
- [ ] Commented out an infrastructure value
- [ ] Added new infrastructure provisioner
- [ ] Added new test config
- [x] Minor input parameter changes
- [ ] Other -> add a comment

NOTE 1:
Current repo owners: @msmshariq, @ThilinaManamgoda, @VimukthiPerera, @chamithkumarage, @kasunbg
Ex repo owners: @yasassri, @sameerawickramasekara, @azinneera, @pasindujw

NOTE 2:
For your job, you can either point to an external testgrid yaml configuration or keep the actual testgrid yaml here within this repo. Usually, people like to keep the testgrid yaml config within their own team repos because its easier to do changes.

You can point to an external testgrid yaml by having the job configration as follows:

$> cat [testgrid-job-configs/Ballerina/bbg-kafka.yaml](https://github.com/wso2/testgrid-job-configs/blob/ce9184d7e1c3719d74ad56325239f54bff21e18b/Ballerina/bbg-kafka.yaml)
```
testgridYamlURL: https://raw.githubusercontent.com/ballerina-platform/ballerina-scenario-tests/scenario-tests/.testgrid-bbg-kafka.yaml

```
More details can be found in user guide - https://docs.google.com/document/d/1fYCUg97VsfoftEo1HfPWjdS6whp4r0noGRzfaxsxmek/edit#

### CODE REVIEW

File: Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# TestGrid Test Configuration Parameters
+version: '0.9'
+emailToList: "bfsi-group@wso2.com,divya@wso2.com,malshani@wso2.com,verushka@wso2.com,kasundh@wso2.com,charithad@wso2.com,akilaa@wso2.com,rivindu@wso2.com,thiloshon@wso2.com"
```
Comment (Dinushi) [inline Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml:3]: Email list needs to be updated. Right ?

Comment (AmilaSamith) [reply Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml:3]: You mean to the following values or else can you suggest?
----------------------------
maheshika@wso2.com,
shariq@wso2.com,
vimukthi@wso2.com,
integration-builder@wso2.com,
iam-builder@wso2.com,
bfsi-group@wso2.com,
ashen@wso2.com,
chanika@wso2.com,
dilini@wso2.com,
divya@wso2.com,
jayangak@wso2.com,
malshani@wso2.com,
mushthaq@wso2.com,
prasanna@wso2.com,
paraparan@wso2.com
-------------------------------

Comment (Akila94) [reply Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml:3]: @AmilaSamith , I think what @Dinushi  means is that the mail "bfsi-group@wso2.com" should be changed to "integration-bfsi-group@wso2.com".

File: Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    description: "ob-scenarios-devtest"
+    file: product-scenarios/test-grid-jobs/test_au/test.sh
+    inputParameters:
+      SolutionVersion: "2.0.0"
+      DeploymentSpec: "AU"
+      OperatingSystem: "Linux"
+      BrowserAutomation.HeadlessEnabled: true
+      InstallGeckodriver: true
+      ApiVersion: "1.3.0"
+      BrowserAutomation.BrowserPreference: "firefox"
```
Comment (Akila94) [inline Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml:46]: Let's add a new line at the end of the file as a practice.

File: Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml (hunk 1 above)
Comment (AmilaSamith) [reply Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml:3]: Thanks for the clarification.I'll make the changes.

File: Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+          inputParameters:
+            InternalPrepareForTest : false
+            parseInfrastructureScript: false
+            KeyPairName: "deployment-prod"
+            CertificateName: "wso2_testgrid"
+            DBUsername: "wso2carbon"
+            DBPassword: "DB_Password"
+            DeploymentSpec: "AU"
+            ElasticSearchRegion: "us-east-1"
+            U2Environment: "Staging"
```
Comment (DivyaPremanantha) [inline Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml:31]: Is this build going to run with staging updates or UAT?

**Review by VimukthiPerera — APPROVED** [review]

File: Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml (hunk 3 above)
Comment (AmilaSamith) [reply Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml:31]: This build is going to run with staging

File: Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml (hunk 1 above)
Comment (AmilaSamith) [reply Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml]: Removed outdated recipients form email list

File: Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  provisioners:
+    - name: distributed-single-node-deployment
+      remoteRepository: "git@github.com:wso2-support/aws-open-banking.git"
+      remoteBranch: "support-2.0.0"
+      description: Provide Infra for running scenario tests
+      scripts:
+        - name: 'prod-wso2ob-200-deployment'
+          description: ''
+          type: CLOUDFORMATION
+          file: obam-with-obkm-au.yaml
```
Comment (SachiniSiriwardene) [inline Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml:21]: change the name of the file to the correct name

**Review by SachiniSiriwardene — APPROVED** [review]

File: Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml (hunk 4 above)
Comment (AmilaSamith) [reply Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml]: correction done [655a665](https://github.com/wso2/testgrid-job-configs/pull/559/commits/655a665f56e6d68e92b908014b08dddbd1fd0aff)

**Review by DivyaPremanantha — APPROVED** [review]

**Pull request merged**